weave/trace_server/clickhouse_trace_server_batched.py

```diff
@@ -4676,6 +4705,10 @@ def _insert_call(self, ch_call: CallCHInsertable) -> None:
     @ddtrace.tracer.wrap(name="clickhouse_trace_server_batched._flush_calls")
     def _flush_calls(self) -> None:
         self._analyze_call_batch_breakdown()
+        if len(self._call_batch) > 0:
+            project_id_idx = ALL_CALL_INSERT_COLUMNS.index("project_id")
+            project_id = self._call_batch[0][project_id_idx]
```

Comment:
while true in practice, there is nothing forcing all calls to have the same project id - I would consider explicitly checking the set

Reply:
This is implemented in less of a hacky way in the actual write path pr.